pages/docs/environments/gitpod.mdx

```diff
@@ -10,10 +10,69 @@ Gitpod is an open-source application that provides a prebuilt development enviro
 
 You can use Gitpod in the cloud. Please note, that Gitpod requires additional configuration to work on the authenticated pages in LinkFree. See instructions for [how to set up your OAuth app](/docs/environments/environment-variables) for more information.
 
-There are 2 ways you can open a repository with Gitpod:
+There are **2** ways you can open a repository with Gitpod.
+
+####1st Method: Prefix the url with `gitpod.io/#`
+
+- Go to the GitHub page of your forked LinkFree repository
```

Comment:
I notice you are using an unordered list in markdown to create steps.  Would it be more clear to have these numbered if they should be completed in order? wdyt?

Reply:
I thought the same. 😉